quimb/tensor/circuit.py

```diff
@@ -408,6 +408,72 @@ def apply_fsim(psi, theta, phi, i, j, parametrize=False, **gate_opts):
     psi.gate_(G, (int(i), int(j)), tags=mtags, **gate_opts)
 
 
+def fsimg_param_gen(params):
+    theta, Zeta, chi, gamma, phi = params[0], params[1],
+    params[2], params[3], params[4]
+
```

Comment:
This doesn't look quite right, shouldn't it be:
```python
theta, Zeta, chi, gamma, phi = params[0], params[1], \
    params[2], params[3], params[4]
```
or 
```python
theta, Zeta, chi, gamma, phi = (
    params[0], params[1], params[2], params[3], params[4]
)
```
i.e. no newline. I'm slightly suprised this is failing the tests, since you are triny to unpack 2 values into 5.

Reply:
You are right. Let me double check.